scripts/run-integration-tests.sh

```diff
@@ -11,12 +11,12 @@ source $DIR/lib/cluster.sh
 
 OS=$(go env GOOS)
 ARCH=$(go env GOARCH)
-AWS_REGION=${AWS_REGION:-us-west-2}
-K8S_VERSION=${K8S_VERSION:-1.14.6}
-PROVISION=${PROVISION:-true}
-DEPROVISION=${DEPROVISION:-true}
-BUILD=${BUILD:-true}
-RUN_CONFORMANCE=${RUN_CONFORMANCE:-false}
+: ${AWS_REGION:=us-west-2}
+: ${K8S_VERSION:=1.14.6}
+: ${PROVISION:=true}
+: ${DEPROVISION:=true}
+: ${BUILD:=true}
+: ${RUN_CONFORMANCE:=false}
```

Comment:
Nice trick!

I had never seen this, found an explanations here https://stackoverflow.com/a/28085062/4992